components/brave_rewards/common/mojom/bat_ledger.mojom

```diff
@@ -1,52 +1,68 @@
 // Copyright (c) 2019 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// You can obtain one at http://mozilla.org/MPL/2.0/.
-module bat_ledger.mojom;
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+module ledger.mojom;
 
 import "brave/components/brave_rewards/common/mojom/ledger.mojom";
 import "brave/components/brave_rewards/common/mojom/ledger_database.mojom";
 import "brave/components/brave_rewards/common/mojom/ledger_types.mojom";
 import "mojo/public/mojom/base/time.mojom";
 import "mojo/public/mojom/base/values.mojom";
 
-interface BatLedgerService {
-  Create(pending_associated_remote<BatLedgerClient> bat_ledger_client,
-         pending_associated_receiver<BatLedger> database) => ();
+interface Ledger {
```

Comment:
I would love to see us move away from the "ledger" terminology. It seems like we are doubling down on it here, though. I understand that since we have "LedgerImpl" (which we can't change yet), it is intuitive to name this "Ledger". However, I think it would be better to go ahead and rename this to something better now (e.g. `RewardsEngine`, but we should throw some ideas around), rather that wait.